### PR TITLE
[ELY-1717] Remove 'wildfly-elytron-credential-source-deprecated'

### PR DESCRIPTION
**Pull request by ashley-abdelsayed98**
https://issues.jboss.org/browse/ELY-1717

### CODE REVIEW

**Comment by fjuma** [comment]
@ashley-abdelsayed98 This one now has some conflicts that need to be resolved.

**Comment by fjuma** [comment]
this is ok to test

**Comment by jamezp** [comment]
this is ok to test

File: wildfly-elytron/pom.xml [hunk 1]
```diff
@@ -915,6 +906,14 @@
                                     <include>org.wildfly.security.x500.cert</include>
                                 </includes>
                                 <excludes>
+                                    <exclude>org.wildfly.security.x500.cert.ElytronMessages_$logger</exclude>
```
Comment (fjuma) [inline wildfly-elytron/pom.xml]: It looks like this exclusion can be removed.

Comment (ashley-abdelsayed98) [reply wildfly-elytron/pom.xml]: removed

**Pull request merged**